Sources/PostgreSQL/Message/PostgreSQLSSLMessage.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  PostgreSQLSSLRequest.swift
+//  Async
+//
+//  Created by franz busch on 15.04.18.
+//
+
+/// First message sent from the frontend during startup if SSL is enabled.
+struct PostgreSQLSSLMessage: Encodable {
+
+    /// The SSL request code. The value is chosen to contain 1234 in the most significant 16 bits,
+    /// and 5679 in the least significant 16 bits.
+    var sslRequestCode: Int32 = 80877103
```

Comment:
Would using hex notation make it more clear how the value is encoded? Also, how about making this value `let`?

Reply:
I just followed the implementation of the startupMessage which is also using the same format and provides a comment above